.kitchen.yml

```diff
@@ -1,19 +0,0 @@
----
```

Comment:
yes, with this being so closely integrated to CFN... until the is a CFN knife driver, you can't really use test kitchen with this.